R/facet-layout.r

```diff
@@ -1,23 +1,24 @@
 # Layout panels in a 2d grid.
 #
 # @params data list of data frames, one for each layer
+# @params env envronment of the plot
 # @params rows variables that form the rows
 # @params cols variables that form the columns
 # @return a data frame with columns \code{PANEL}, \code{ROW} and \code{COL},
 #   that match the facetting variable values up with their position in the
 #   grid
-layout_grid <- function(data, rows = NULL, cols = NULL, margins = NULL,
+layout_grid <- function(data, env, rows = NULL, cols = NULL, margins = NULL,
```

Comment:
It would be slightly better to avoid changing the API for this function by putting `env` at the end, and setting a default value (`parent.frame()` would be fine).